components/src/main/java/uk/gov/hmrc/components/molecule/item/MultiColumnRowView.kt

```diff
@@ -173,6 +174,27 @@ class MultiColumnRowView @JvmOverloads constructor(
         }
     }
 
+    fun setText1ClickAction(listener: OnClickListener, clickDescription: CharSequence? = null) {
```

Comment:
Can we switch the order around of these so the listener is at the end? This will allow us do do things like this in Kotlin:
`setText1ClickAction { Toast.makeText(context, "text 1 custom action invoked", LENGTH_LONG).show() }`

Reply:
Good suggestion! Changed in https://github.com/hmrc/android-components/pull/41/commits/69a87ddd2cebf6fcbab6af997ae05d35b2fcbd17 :)